README.md

```diff
@@ -49,14 +49,25 @@ The OCI and OCM support can be found in packages
 
 Install the latest release with
 
-- [Bash](#bash)
-- [Homebrew](#homebrew)
-- [NixOS](#nixos)
-- [AUR](#aur)
-- [Container](#container)
-- [Chocolatey](#chocolatey)
-- [Winget](#winget)
-- [GitHub Releases](https://github.com/open-component-model/ocm/releases/latest)
+- [Open Component Model](#open-component-model)
+  - [OCM Specifications](#ocm-specifications)
+  - [OCM Library](#ocm-library)
+  - [OCM CLI](#ocm-cli)
+  - [Installation](#installation)
+    - [Bash](#bash)
+    - [Homebrew](#homebrew)
+    - [NixOS](#nixos)
+    - [AUR](#aur)
+    - [Container](#container)
+      - [Build and run it yourself](#build-and-run-it-yourself)
+    - [Chocolatey](#chocolatey)
+    - [Winget](#winget)
+  - [Examples](#examples)
+  - [GPG Public Key](#gpg-public-key)
+  - [Contributing](#contributing)
+  - [Release Process](#release-process)
+  - [Licensing](#licensing)
+    - [Third Party Licenses](#third-party-licenses)
```

Comment:
this TOC does not fit to the section: `Install the latest release with` - @morri-son can you revert that change?

Reply:
corrected and also added a new line to the TOC. I guess the TOC has been updated on save and since this is a TOC for just a sub part of the doc, it also added the sections before and after...